### PR TITLE
Qt: Implement Wiimote Refresh button

### PR DESCRIPTION
**Pull request by spycrab**
Small PR that adds a minor feature that I previously forgot to implement.
Screenshot

### CODE REVIEW

**Comment by leoetlino** [comment]
Do you have a screenshot for this?

File: Source/Core/DolphinQt2/Config/ControllersWindow.cpp [hunk 1]
```diff
@@ -115,23 +115,32 @@ void ControllersWindow::CreateGamecubeLayout()
...
+static QHBoxLayout* CreateSubItem(QWidget* label, QWidget* widget)
 {
   QHBoxLayout* hbox = new QHBoxLayout();
   hbox->addItem(new QSpacerItem(25, 1));
 
   if (label != nullptr)
     hbox->addWidget(label);
 
-  hbox->addWidget(widget);
+  if (hbox != nullptr)
```
Comment (lioncash) [inline Source/Core/DolphinQt2/Config/ControllersWindow.cpp]: Why would hbox be null here?

File: Source/Core/DolphinQt2/Config/ControllersWindow.cpp [hunk 2]
```diff
@@ -115,23 +115,32 @@ void ControllersWindow::CreateGamecubeLayout()
...
 {
-  QLabel* label = text.isEmpty() ? nullptr : new QLabel(text);
-  return CreateSubItem(label, widget);
+  QHBoxLayout* hbox = new QHBoxLayout();
+  hbox->addItem(new QSpacerItem(25, 1));
+
+  if (label != nullptr)
+    hbox->addWidget(label);
+
+  if (hbox != nullptr)
```
Comment (lioncash) [inline Source/Core/DolphinQt2/Config/ControllersWindow.cpp]: ditto.

**Comment by lioncash** [comment]
Unrelated to the PR, but I should have probably thought of asking this before the dialog itself was created, but does the setting dialog window still look fine on Hi-DPI displays? This used to be an issue with wx.

**Comment by spycrab** [comment]
@lioncash Should look fine as there are no fixed constraints put on the Window or its content (Fixed px sizes, etc). I don't have such a Display, so I can't test it but I'm 99% sure that there won't be any issues there.

**Comment by Magmarock** [comment]
How do you get that dark theme?

**Comment by spycrab** [comment]
@Magmarock This is off-topic and shouldn't be discussed here.

**Comment by Magmarock** [comment]
@spycrab Please accept my apologies. I'm new to this site and not sure how to send private message.

**Comment by Magmarock** [comment]
Can I reach you through this email?

**Pull request merged**